opal/mca/hwloc/hwloc191/hwloc/config/hwloc.m4

```diff
@@ -1306,8 +1308,8 @@ AC_DEFUN([_HWLOC_CHECK_DECL], [
   AC_REQUIRE([AC_PROG_CC])
   AC_COMPILE_IFELSE([AC_LANG_PROGRAM(
        [AC_INCLUDES_DEFAULT([$4])
-       $1(int,long,int,long,int,long,int,long,int,long);],
-       [$1(1,2,3,4,5,6,7,8,9,10);])],
+       void * $1;],
+       )],
```

Comment:
@ggouaillardet Can you back this out / squash the removal into the commit that added it?